api-module-library/frontify/api.js

```diff
@@ -0,0 +1,141 @@
+const { get } = require('@friggframework/assertions');
+const { OAuth2Requester } = require('@friggframework/module-plugin');
+const querystring = require('querystring');
+
+class Api extends OAuth2Requester {
+    constructor(params) {
+        super(params);
+        this.domain = get(params, 'domain', null);
+
+        if (this.domain) {
+            this.baseUrl = `https://${this.domain}/graphql`;
+            this.tokenUri = `https://${this.domain}/api/oauth/accesstoken`;
+        }
+    }
+
+    setDomain(domain) {
```

Comment:
I'm not sure @leofmds , that could mean that we'd have to start to validate inputs in other methods as well? Since only few people will be working with this module we can assume that they will know what they are doing IMO :crossed_fingers:

Reply:
@roboli I believe this is fine by now, but we should expect that developers who are going to use these integrations are the end users, and we should facilitate the usage. As opposed to a developer passing a full URL, and then having to jump into code to see the right way of passing it. An error message would already indicate what needs to be changed.